src/D2L.Bmx/Okta/OktaApi.cs

```diff
@@ -142,6 +143,24 @@ internal class OktaApi : IOktaApi {
 		throw new BmxException( "Account could not be found" );
 	}
 
+	async Task<string?> IOktaApi.GetMeResponseAsync( string sessionId ) {
+
+		try {
+			using var meResponse = await _httpClient.GetAsync( "users/me" );
+			if( !meResponse.IsSuccessStatusCode ) {
+				return null;
+			}
+
+			var meJson = await meResponse.Content.ReadAsStringAsync();
+			var me = await meResponse.Content.ReadFromJsonAsync( SourceGenerationContext.Default.OktaMeResponse );
+			return me?.Id ?? null;
```

Comment:
this should be sufficient
```suggestion
			return me?.Id;
```